iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -97,6 +107,9 @@ private async Task InitializeAndOpenClientAsync()
 
                     s_deviceClient = DeviceClient.CreateFromConnectionString(_deviceConnectionStrings.First(), _transportType, _clientOptions);
                     s_deviceClient.SetConnectionStatusChangesHandler(ConnectionStatusChangeHandler);
+
+                    // Updating the client's operation timeout to 30secs for the purpose of a sample. The default value is 4mins.
+                    s_deviceClient.OperationTimeoutInMilliseconds = (uint)s_deviceOperationTimeout.TotalMilliseconds;
```

Comment:
this is to encourage a faster timeout in case a persistent retryable exception is encountered